blockchain_api/artifacts/src/chaincode/opetbot/opetbot.go

```diff
@@ -173,6 +193,12 @@ func (t *OpetCode) createDocument(APIstub shim.ChaincodeStubInterface, args []st
     return shim.Success(nil)
 }
 
+
+/*
+    The function retrieves document by it's uid and user's uid. Checks that the user has the document.
```

Comment:
retrieveDocument returns the document by it's uid and user's uid.
Checks that the user owns the requested document.